CMakeLists.txt

```diff
@@ -42,23 +42,27 @@ option("BUNDLE_SKIP_FCKIT" "Don't build fckit" "ON") # Skip fckit build unless u
 option("BUNDLE_SKIP_ATLAS" "Don't build atlas" "ON") # Skip atlas build unless user passes -DBUNDLE_SKIP_ATLAS=OFF
 
 ecbuild_bundle( PROJECT eckit GIT "https://github.com/ecmwf/eckit.git" TAG 1.20.2 )
-ecbuild_bundle( PROJECT fckit GIT "https://github.com/ecmwf/fckit.git" TAG 0.9.5 )
-ecbuild_bundle( PROJECT atlas GIT "https://github.com/ecmwf/atlas.git" TAG 0.31.1 )
+ecbuild_bundle( PROJECT fckit GIT "https://github.com/ecmwf/fckit.git" TAG 0.10.0 )
+ecbuild_bundle( PROJECT atlas GIT "https://github.com/ecmwf/atlas.git" TAG 0.32.1 )
 
 # External FMS
 find_package(FMS 2022.04 REQUIRED COMPONENTS R4 R8)
 
+# Core JEDI repositories
+# ----------------------
+ecbuild_bundle( PROJECT oops  GIT "https://github.com/jcsda-internal/oops.git"  TAG 3318acc4b527e49fcc2e2d78549a06fc4ab43dd0 ) # develop on apr 18
+ecbuild_bundle( PROJECT vader GIT "https://github.com/jcsda-internal/vader.git" TAG c5e0c1a8d4feeedacfef788f7670511e4db41d5d ) # develop on apr 18
+ecbuild_bundle( PROJECT saber GIT "https://github.com/jcsda-internal/saber.git" TAG fc8992d7c6634c384f2f61eb121743ab8ede0d6a ) # develop on apr 18
+
+
 if(UFS_APP MATCHES "^(ATMAERO)$" OR UFS_APP MATCHES "^(S2S)$" OR UFS_APP MATCHES "^(NG-GODAS)$")
-  ecbuild_bundle( PROJECT gsw GIT "https://github.com/jcsda-internal/GSW-Fortran.git" TAG 1a02ebaf6f7a4e9f2c2d2dd973fb050e697bcc74 )  # develop on jan 28
+  ecbuild_bundle( PROJECT gsw GIT "https://github.com/jcsda-internal/GSW-Fortran.git" TAG 1a02ebaf6f7a4e9f2c2d2dd973fb050e697bcc74 )  # develop on apr 18
 endif()
 
-# Core JEDI repositories
-# ----------------------
-ecbuild_bundle( PROJECT oops  GIT "https://github.com/jcsda-internal/oops.git"  TAG 0932772f4b20b0defedfb12e31a0c17c18caac71 ) # develop on feb 1
-ecbuild_bundle( PROJECT vader GIT "https://github.com/jcsda-internal/vader.git" TAG 3c8fc58ed67c013a955ca7d8f8d2346f2844ac96 ) # develop on jan 28
-ecbuild_bundle( PROJECT saber GIT "https://github.com/jcsda-internal/saber.git" TAG 1203356523b123654a8871846a55a06ed26b7247 ) # develop on jan 28
-ecbuild_bundle( PROJECT ioda  GIT "https://github.com/jcsda-internal/ioda.git"  TAG 39ab4dd1dfe737f71385af802c2b5c67c2ef8efc ) # develop on jan 28
-ecbuild_bundle( PROJECT ufo   GIT "https://github.com/jcsda-internal/ufo.git"   TAG ab106a84c770414a3edf9dacabeadea5d70d9edf ) # develop on jan 28
+ecbuild_bundle( PROJECT crtm  GIT "https://github.com/jcsda-internal/crtm.git"  TAG v2.4-jedi.2 )
```

Comment:
Should crtm be built here? When I loaded it as a module I couldn't get it to work, but it might have been an error on my end

Reply:
I didn't realize that ... will remove the line and try again

Reply:
No, you don't need it, builds just fine with v2.4-jedi.2 from spack-stack

Reply:
I had issues with faling tests

Reply:
which tests?

Reply:
I've got only these `fv3jedi_*` test failures, but keep in mind that I am using gsibec@1.1.2 which is the reason for some of them to fail:
```
	1189 - fv3jedi_test_tier1_model_ufs_coldstart (Failed)
	1194 - fv3jedi_test_tier1_forecast_ufs_coldstart (Failed)
	1214 - fv3jedi_test_tier1_dirac_geos_gsi_global (Failed)
	1215 - fv3jedi_test_tier1_3dvar_geos_sondes (Failed)
	1216 - fv3jedi_test_tier1_3dvar_geos_ozone (Failed)
	1217 - fv3jedi_test_tier1_dirac_gfs_gsi_global (Failed)
	1218 - fv3jedi_test_tier1_3dvar_gfs_sondes (Failed)
	1236 - fv3jedi_test_tier1_4denvar (Failed)
```

Reply:
I get many many segfaults after successfully completing the tests for `qg` like this one:
```
...
171: OOPS_STATS Run end                                  - Runtime:      2.57 sec,  Memory: total:    26.99 Mb, per task: min =    26.99 Mb, max =    26.99 Mb
171: Run: Finishing oops::Forecast<QG> with status = 0
171: [TestReference] Comparison is done
171: OOPS Ending   2023-04-19 14:33:36 (UTC-0600)
3/3 Test #171: test_qg_truth_pert_heat ..........***Exception: SegFault  2.92 sec
```
But I think this is a mac specific issue that has to do with the openmp library

Reply:
So yes, please remove the crtm line here